pages/DSNP/DSNP-Messages.md

```diff
@@ -62,19 +62,27 @@ a public post (was Announcement)
 
 | field     | description | type |
 |-------    |-------------| ----|
-| inReplyTo | messageID replied to | bytes32
+| dnspRoute (fromAddress) | id of the sender | bytes20
 | messageID | keccak-256 hash of content stored at uri |  bytes32
 | uri       | content uri | bytes
- 
 
-**NOTE** If origin broadcasts and replies stay the same type, inReplyTo is allowed to be blank.  If replies are separated into their own type, the inReplyTo field here will be dropped.
+#### Reply
+a public reply post
+
+| field     | description | type |
+|-------    |-------------| ----|
+| dnspRoute (inReplyTo) | messageID replied to | bytes32
+| messageID | keccak-256 hash of content stored at uri |  bytes32
+| fromAddress | sender's identity | bytes20
```

Comment:
`fromAddress` isn't needed in the actual DSNP data segment.

Reply:
I think it is. Where else would that data come from?

Reply:
the fromAddress of the message?

Reply:
That assumes that the message is coming from the identity contract. If this is a separate contract, then it doesn't work. I think I added it to all the ones that need it. Most of then actually don't. (Drop for instance)